.github/workflows/packages.yml

```diff
@@ -99,8 +101,23 @@ jobs:
                 working-directory: "src/Sylius/${{ matrix.package }}"
 
             -
-                name: Test package
-                run: bin/test-package src/Sylius/${{ matrix.package }}
+                name: Setup Database(for ApiBundle only)
+                working-directory: "src/Sylius/${{ matrix.package }}"
+                run: test/bin/console doctrine:schema:update --force -e test
+                if: always() && matrix.package == 'Bundle/ApiBundle'
+
+            -
+                name: Run PHPSpec
+                working-directory: "src/Sylius/${{ matrix.package }}"
+                run: vendor/bin/phpspec run --ansi --no-interaction -f dot
+
+            -
+                name: Run PHPUnit
+                working-directory: "src/Sylius/${{ matrix.package }}"
+                run: |
+                    if [[ -e "phpunit.xml.dist" && -e "phpunit.xml" ]]; then
```

Comment:
```suggestion
                    if [[ -e "phpunit.xml.dist" || -e "phpunit.xml" ]]; then
```

Reply:
Can we somehow move it to the job's `if` statement?

Reply:
It should say that the job was skipped if the condition is not met, because right now no unit tests ran but the status is all green.

Reply:
I think you can skip the condition altogether, I tried running phpunit in a component without the config files and it didn't fail.

Reply:
Never mind, the components won't even have `vendor/bin/phpunit`, because it's not a requirement 😅